HeterogeneousCore/SonicCore/src/SonicClientBase.cc

```diff
@@ -40,19 +59,39 @@ void SonicClientBase::start(edm::WaitingTaskWithArenaHolder holder) {
   holder_ = std::move(holder);
 }
 
-void SonicClientBase::start() { tries_ = 0; }
+void SonicClientBase::start() { 
+    tries_ = 0; 
```

Comment:
no longer used elsewhere? though it may be useful to keep track of how many RetryActions are attempted for a given call (for logging purposes)